dotnet/Razorvine.Pyrolite/Benchmarks/PicklerBenchmarks.cs

```diff
@@ -0,0 +1,69 @@
+﻿using BenchmarkDotNet.Attributes;
+using Razorvine.Pickle;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Benchmarks
+{
+    [GenericTypeArguments(typeof(string))]
+    [GenericTypeArguments(typeof(double))]
+    [GenericTypeArguments(typeof(int))]
+    [GenericTypeArguments(typeof(bool))]
+    public class PicklerBenchmarks<T>
+    {
+        [Params(1000)]
```

Comment:
Did you want to have multiple params here? Or is `1000` enough?

Reply:
1000 should be enough. If I add one more test case the time to run the benchmarks is going to double